autogpts/autogpt/autogpt/commands/image_gen.py

```diff
@@ -102,7 +102,7 @@ def generate_image_with_hf(prompt: str, output_file: Path, agent: Agent) -> str:
                 break
         else:
             try:
-                error = json.loads(response.text)
+                error = json_loads(response.text)
```

Comment:
do you have a reasonable suspicion that `response.text` might ever be dirty JSON?

Reply:
No, I'll revert that 😅